core/worker.go

```diff
@@ -2338,29 +2362,27 @@ func (w *worker) FinalizeAssemble(chain consensus.ChainHeaderReader, newWo *type
 // AddPendingBlockBody adds an entry in the lru cache for the given pendingBodyKey
 // maps it to body.
 func (w *worker) AddPendingWorkObjectBody(wo *types.WorkObject) {
-	// do not include the tx hash while storing the body
-	woHeaderCopy := types.CopyWorkObjectHeader(wo.WorkObjectHeader())
-	woHeaderCopy.SetTxHash(common.Hash{})
```

Comment:
this was done so that we can support any number of txs broadcast set without needing to make a copy for a combination of workobject with a tx set

Reply:
as long as the workers cache can handle large amounts of workshares, this is a fine thing